src/Microsoft.AspNetCore.Authentication.OpenIdConnect/OpenIdConnectExtensions.cs

```diff
@@ -18,9 +18,12 @@ public static AuthenticationBuilder AddOpenIdConnect(this AuthenticationBuilder
             => builder.AddOpenIdConnect(OpenIdConnectDefaults.AuthenticationScheme, configureOptions);
 
         public static AuthenticationBuilder AddOpenIdConnect(this AuthenticationBuilder builder, string authenticationScheme, Action<OpenIdConnectOptions> configureOptions)
+            => builder.AddOpenIdConnect(authenticationScheme, OpenIdConnectDefaults.Caption, configureOptions);
```

Comment:
Haha, wasn't this constant dead code? :sweat_smile: 

It would be nice to rename it to `DisplayName` and add similar constants in the other providers, for consistency.